sdk/python/test/e2e/utils.py

```diff
@@ -47,7 +47,8 @@ def verify_job_e2e(
 
     # Job should have Created, Running, and Succeeded conditions.
     conditions = client.get_job_conditions(job=job)
-    if len(conditions) != 3:
+    # If Job is complete fast, it has 2 conditions: Created and Succeeded.
+    if len(conditions) != 3 and len(conditions) != 2:
```

Comment:
This looks bit odd. Can we clean this check?

Reply:
@johnugeorge Do you want to remove this check ?
I noticed that PyTorchJob has just 2 conditions (e.g. Created and Succeeded), if you run it with image that executes very fast. E.g. `docker.io/hello-world`.

Reply:
Isn't that a bug which needs to be resolved separately?

Reply:
Probably. @tenzen-y @kuizhiqing @tenzen-y What are your thoughts here ?
The main problem is that when reconciliation loop starts, the training Pod is already Succeeded, so we didn't add Running status for PyTorchJob.